src/status_im/utils/utils.cljs

```diff
@@ -20,11 +20,11 @@
    (.alert (.-Alert rn-dependencies/react-native)
            title
            content
-           ; Styles are only relevant on iOS. On Android first button is 'neutral' and second is 'positive'
+           ;; Styles are only relevant on iOS. On Android first button is 'neutral' and second is 'positive'
```

Comment:
Almost everywhere in the codebase, comments were preceded by double-semicolon, that's the only reason for this change :)
We should definitely agree on some convention.